addons/stock/stock.py

```diff
@@ -2068,6 +2068,17 @@ def onchange_product_id(self, cr, uid, ids, prod_id=False, loc_id=False, loc_des
             result['location_dest_id'] = loc_dest_id
         return {'value': result}
 
+    def _prepare_picking_assign(self, cr, uid, move, context=None):
+        """This method is designed to be inherited"""
```

Comment:
Maybe a better name also?  """Prepares a new picking for this move as it could not be assigned to another picking"""